### DISCUSSION TITLE
Different structure of out-file?!

### DISCUSSION THREAD

**Discussion by BastianStahra** [firemodels/fds#13349] [Q&A] [answered]
Hello, I have a question regarding the out-files of fds.
I get different out-files depending on 
- the version of fds and 
- on the machine I simulate on. 

Can anybody help me why that is? Can not find an explanation in the User or TR-Guide

Below the comparison of simulation on a Workstation (Windows) and the PLEIADES (Linux)

Older version 6.7.9
![image](https://github.com/user-attachments/assets/784cf05e-246d-4dd7-8195-29eae3f7c016)

Both versions give the CFL-number and divergence liek this:
  Mesh   20
       Max CFL number:  0.50E+00 at (29,39,79)
       Max divergence:  0.16E+02 at (27,39,43)
       Min divergence: -0.41E+01 at (27,39,44)
       Max VN number:   0.26E+00 at (26,40,28)
       Total Heat Release Rate:          11922.565 kW
       Radiation Loss to Boundaries:     -8232.543 kW

Newer version 6.9.1
![image](https://github.com/user-attachments/assets/d3cf42b4-03be-4a5f-a5bc-d68e1274e78b)

In the newer version I don't get the CFL and divergence from the PLEIADES, but I get this other diagnostic-output.

Can someone explain that to me? 

I am very grateful for an answer.

AMD-Workstation

**Comment by Er9y714** [comment, marked as answer]
I think `VERBOSE `and `SUPPRESS_DIAGNOSTICS `parameters in User guide will help you.

**Comment by mcgratta** [comment]
When the job exceeds 32 meshes, FDS switches to less verbose diagnostic output unless you say otherwise. As for different output in different versions, sure, we sometimes change the output as needed.

**Comment by BastianStahra** [comment]
Thank you very much!
